dvc.yaml

```diff
@@ -6,18 +6,34 @@ stages:
         outs:
         - ${data.root}/${item}_${data.index_file}
         deps:
-        - ${data.root}/${item}.zip
+        - ${data.root}/${item}/${item}.zip
```

Comment:
@ykasimov do you really need this data as dependency? For large datasets it takes a lot of time to calculate hashes. Also,  it seems unnecessary to add this to  DVC storage.

Reply:
you are right, thank you, i'll remove it 👍